Make.inc

```diff
@@ -381,6 +381,26 @@ else
 build_shlibdir = $(build_libdir)
 endif
 
+# If we are running on ARM, set certain options automatically
+ifneq (,$(findstring arm,$(ARCH)))
+LLVM_ASSERTIONS=1
+LLVM_FLAGS+="--with-cpu=cortex-a9 --with-float=hard --with-abi=aapcs-vfp --enable-targets=arm --enable-optimized --enable-assertions"
```

Comment:
For the time being, I think we should leave the assertions in as the ARM builds are running into a lot of problems right now.  Additionally, this doesn't really change the user experience much, as our documentation at the moment [suggests a course of action](https://github.com/JuliaLang/julia/blob/1b2f4fa9d75c604978920153388bc9bd62e3dc6c/README.arm.md#arm-specific-makeuser) that has the same effect as this PR.  I'll take out `--enable-optimized` and `--enable-targets` though.  Note that `--enable-assertions` is [apparently the default](http://llvm.org/docs/BuildingLLVMWithAutotools.html#a-quick-summary).